pkg/openstack/models/models.go

```diff
@@ -151,4 +202,10 @@ func init() {
 	registry.ModelRegistry.MustRegister("openstack:model:subnet", &Subnet{})
 	registry.ModelRegistry.MustRegister("openstack:model:floating_ip", &FloatingIP{})
 	registry.ModelRegistry.MustRegister("openstack:model:project", &Project{})
+	registry.ModelRegistry.MustRegister("openstack:model:subnet_to_network", &SubnetToNetwork{})
```

Comment:
Model names don't have the `link_` prefix like the other data sources. Please add the prefix to the link models.